webapp/src/main/java/com/box/l10n/mojito/boxsdk/BoxSDKServiceConfigProvider.java

```diff
@@ -1,16 +1,22 @@
 package com.box.l10n.mojito.boxsdk;
 
-import com.box.l10n.mojito.service.boxsdk.BoxSDKServiceConfigEntityService;
-import org.springframework.beans.factory.annotation.Autowired;
+import com.box.l10n.mojito.service.boxsdk.BoxSDKServiceConfigEntityRepository;
 import org.springframework.stereotype.Component;
 
 /** @author wyau */
 @Component
 public class BoxSDKServiceConfigProvider {
 
-  @Autowired BoxSDKServiceConfigFromProperties boxSDKServiceConfigFromProperties;
+  private final BoxSDKServiceConfigFromProperties boxSDKServiceConfigFromProperties;
 
-  @Autowired BoxSDKServiceConfigEntityService boxSDKServiceConfigEntityService;
+  private final BoxSDKServiceConfigEntityRepository boxSDKServiceConfigEntityRepository;
```

Comment:
changing from service to repository, to fix one of the circular dependency issue.